components/ILIAS/LearningSequence/PRIVACY.md

```diff
@@ -0,0 +1,63 @@
+# Learning Sequence Privacy
+
+Disclaimer: This documentation does not warrant completeness or correctness. Please report any missing or wrong
+information using the [ILIAS issue tracker](https://mantis.ilias.de) or contribute a fix
+via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+### General Information
+
+The Learning Sequence leads users through multiple pieces of content in a row. It reacts to the Learning Progress of its child objects and can summarize them into one combined Learning Progress. Therefor, the Learning Sequence mostly contains references but may display a snapshot of a user's learning path and hold the final evaluation as one resulting Learning Progress.
+
+As of the time of writing, Users do not need to be Members to access and progress in the Learning Sequence's content by default. If a child object is configured in a way that Users have access, the Users still have access when it's dropped in to the Learning Sequence. This may put the privacy and access concerns primarily onto the child objects. Only with further configuration and permission management the Learning Sequence can serve as a gate for controlling access. Alternatively, a Learning Sequence may be placed into a Course and use its features for access control.
+
+### Integrated Services
+
+- The Learning Sequence component employs the following services, please consult the respective privacy.mds:
+    - [Metadata](../MetaData/Privacy.md)
+    - [AccessControl](../AccessControl/PRIVACY.md)
+    - [Object](../ILIASObject/PRIVACY.md)
+    - [InfoScreen](../InfoScreen/PRIVACY.md)
+    - [Mail](../Mail/PRIVACY.md) for notifications
+    - [Tracking](../Tracking/PRIVACY.md) may control if a user can progress the sequence and the sequence has its own Learning Progress as well
+    - [Membership](../Membership/PRIVACY.md)
+- Most of the actual data storage is not done by the Learning Sequence object itself. Instead the services Membership and Tracking are heavily used.
+
+## Data being stored
+
+- **Participants (Members)**: Adding users as Participants (Members) to the Learning Sequence references their User object by ID. A User may not be a Member but still create and modify a Learning Progress status for any of the child objects they can access.
```

Comment:
the 2nd sentence here, just for me to clarify: non-Members information (Learning Progress, ID etc.) is not stored here, is it? 
**If it's not:** you can delete this sentence (I think this information is already described in General Information, line 11,  and does not belong here)

Reply:
changed